controller/rancher/rancher.go

```diff
@@ -90,10 +102,21 @@ type RMetaFetcher struct {
 type CertificateFetcher interface {
```

Comment:
Wonder if we can move CertificateFetcher code to its own file under rancher package? Otherwise rancher.go gets too bulky